CONTRIBUTING.md

```diff
@@ -189,14 +194,22 @@ git pull upstream develop
 
 Be sure to create your PR against the **develop** branch!
 
-Put a link to the PivotTracker story this addresses so that we keep the connection between the two systems (PivotalTracker and GitHub).  
+We connect the PR in GitHub to the PivotalTracker story manually (simple, but quick and explicit):
+- put a link to the PivotalTracker task in the PR description, _and_
```

Comment:
We refer to "story" in the line above, and now switch to "task".   Suggest we use "story" throughout.